apps/web/src/components/Navbar/index.tsx

```diff
@@ -36,16 +37,38 @@ const Navbar = () => {
           </a>
         </Link>
 
+        {isLoggedIn && (
```

Comment:
Add create project inside this condition, to avoid repetition.

Reply:
done